nav2_system_tests/src/system/tester_node.py

```diff
@@ -329,6 +405,16 @@ def run_all_tests(robot_tester):
         result = test_RobotMovesToGoal(robot_tester)
 
     # Add more tests here if desired
+    if (result):
+        result = test_RobotGetsBadBehaviorTree(robot_tester)
+
+    time.sleep(5)
```

Comment:
I had very strange errors here.. I think there exist already a PR/issue somewhere that's about complaining that we cannot "stress test" the bt_navigator with new goals or something.. Might also be a DDS specific error.. Actually seeing quite a few here with the new Callback (but very sporadic.. Once existing, 3 times not. This in general was reported also by others that local tests might fail sometimes and pass on CI..)